README.md

```diff
@@ -0,0 +1,146 @@
+
+<!-- TODO: update when connector will be published -->
+- [See the listing in the Hasura Hub](https://hasura.io/connectors/singlestore)
+- [Hasura V3 Documentation](https://hasura.io/docs/3.0)
+
+## Features
+
+The following matrix lists the features supported by the Hasura SingleStore connector:
+
+| Feature                         | Supported | Notes |
+| ------------------------------- | --------- | ----- |
+| Native Queries + Logical Models | ❌     |       |
+| Simple Object Query             | ✅     |       |
+| Filter / Search                 | ✅     |       |
+| Simple Aggregation              | ✅     |       |
+| Sort                            | ✅     |       |
+| Paginate                        | ✅     |       |
+| Table Relationships             | ✅     |       |
+| Views                           | ✅     |       |
+| Distinct                        | ✅     |       |
+| Remote Relationships            | ✅     |       |
+| Mutations                       | ❌     | coming soon       |
+
+## Prerequisites
+
+Ensure that the following prerequisites are met before using the connector:
+
+1. Install [DDN CLI](https://hasura.io/docs/3.0/cli/installation) and [Docker](https://docs.docker.com/engine/install/).
+2. A [supergraph](https://hasura.io/docs/3.0/getting-started/init-supergraph)
+3. A [subgraph](https://hasura.io/docs/3.0/getting-started/init-subgraph)
+4. An active [SingleStore](https://www.singlestore.com/) deployment that serves as the data source for the API.
+
+
+## Using the SingleStore connector
+
+The following steps explain how to initialize and configure the connector for local developments. For information on deploying a connector after it has been configured, refer to [Deploy a Connector](https://hasura.io/docs/3.0/getting-started/deployment/deploy-a-connector).
+<!-- TODO: test steps after the connector will be published to hub -->
+### Step 1: Authenticate your CLI session
+
+```bash
+ddn auth login
+```
+
+### Step 2: Configure the connector
+
+Once you have an initialized supergraph and subgraph, run the initialization command in interactive mode while 
+providing a name for the connector in the prompt:
+
+```bash
+ddn connector init <connector-name>  -i
+```
+
+#### Step 2.1: Choose the singlestore from the list
+
+#### Step 2.2: Choose a port for the connector
+The CLI will ask for a specific port to run the connector on. Choose a port that is not already in use or use the 
+default suggested port.
+
+#### Step 2.3: Provide the env vars for the connector
+
+Specify the connection string used to connect to the SingleStore database using the `SINGLESTORE_URL` variable as a key-value pair in this file. The connection string must be in the `mysql://[<username>[:<password>]][@<host>:[<port>]]/[<database>][?<key1>=<value1>[&<key2>=<value2>]]` format. 
```

Comment:
Just checked with other connectors. With the newer version of CLI, there is no need to modify the file. 
It will automatically prompt the user to set env variable. Will delete this part